lib/pymedphys/_experimental/streamlit/apps/iview_to_dicom.py

```diff
@@ -72,19 +144,60 @@ def _create_portal_image_dicom_dataset(
 
     pixel_spacing = 1 / pixels_per_mm * sid / sad
 
+    date = datetime.strftime("%Y%m%d")
+    time = datetime.strftime("%H%M%S.%f")
+
+    # Many of the below was input without a full understanding of the
+    # implications of the choices being made.
     ds = _pp_dcm_create.dicom_dataset_from_dict(
         {
-            "ImageType": ["ORIGINAL", "PRIMARY", "PORTAL"],
+            "Modality": "RTIMAGE",
+            "InstanceCreatorUID": PYMEDPHYS_ROOT_UID,
+            "SOPClassUID": "1.2.840.10008.5.1.4.1.1.481.1",
+            "SOPInstanceUID": pydicom.uid.generate_uid(prefix=f"{PYMEDPHYS_ROOT_UID}."),
+            # From <https://dicom.innolitics.com/ciods/cr-image/general-image/00080008>
+            #
+            # * is the image an ORIGINAL Image; an image whose pixel
+            #   values are based on original or source data
+            # * is the image a SECONDARY Image; an image created after
+            #   the initial patient examination
+            "ImageType": ["ORIGINAL", "SECONDARY", "PORTAL"],
+            # Purposefully avoided all concepts of 'Study' if possible
+            # as this would require appropriately grouping the images
+            # from each field. Certainly doable, and arguably it should
+            # be done, but out of scope for the current task.
+            "PatientID": patient_id,
```

Comment:
OK, this addresses a comment I made elsewhere.
Very good, happy to see it

Reply:
> OK, this addresses a comment I made elsewhere.

Do you mean this comment:

https://github.com/pymedphys/pymedphys/pull/1387#issuecomment-774913098

Does that mean the `base` branch can be approved and merged?

Reply:
Yes.
I think not providing the UID hierarchy early on in the construction 
is a mistake but it is your mistake to make.

Reply:
Kk... if you feel strongly about this, I can see I have can expand on it appropriately on Thursday?